plugins/autostart/permissions/default.toml

```diff
@@ -0,0 +1,19 @@
+"$schema" = "schemas/schema.json"
+[default]
+description = """
+This permission set configures if your
+application can enable or disable auto
+starting the application on boot.
+
+#### Granted Permissions
+
+It allows all to check, enable and
+disable the automatic start on boot.
+
+"""
```

Comment:
this multiline description is messing up the generated table, see https://github.com/tillmann-crabnebula/plugins-workspace/blob/feat/default-permissions/plugins/autostart/permissions/autogenerated/reference.md, not sure if we can fix that or maybe keep the description to a single line?

Reply:
I actually have some code locally that fixes this by wrapping the markdown into an html table but would address this in another PR.

Reply:
For reference https://github.com/tauri-apps/tauri/pull/10113

Reply:
Should we wait for 10113 before we merge this one here then?

Reply:
I think it does not matter too much as the generated markdown is not consumed anywhere yet.